src/main/java/edu/bu/metcs673/trackr/controller/BankAccountController.java

```diff
@@ -87,4 +84,28 @@ public ResponseEntity<GenericApiResponse> modifyBankAccount(@PathVariable(value
                         MessageFormat.format(CommonConstants.DEACTIVATE_BANK_ACCOUNT, String.valueOf(id))),
                 HttpStatus.OK);
     }
+    @GetMapping("/find/{userId}/{bankAccountId}")
+    public ResponseEntity<GenericApiResponse1<BankAccount>> findBankAccountById(@PathVariable(value = "userId") long userId,
```

Comment:
The generic response additional data field is not meant to be a pass through for returning data. We need to create a specific response shape for returning data.